utils/utils.go

```diff
@@ -2,67 +2,28 @@ package utils
 
 import (
 	"errors"
-	"golang.org/x/crypto/bcrypt"
+	"fmt"
 	"regexp"
 	"strconv"
 	"time"
 )
 
-var emailRegex = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
-
+func StringtoTime(input string) (time.Time, error) {
```

Comment:
now that this func has been reduced to single line, there is no need for a separate function for this just feel free to use time.Parse() directly where needed